neuraxle/base.py

```diff
@@ -3621,6 +3648,108 @@ def load(self, context: ExecutionContext, full_dump=True) -> BaseStep:
         return loaded_self.load(context, full_dump)
 
 
+def assert_has_services(has_service_assertions: List[Type], context: ExecutionContext):
```

Comment:
Agreed to the name change. As for the move, it could be placed in the LocalServiceAssertionWrapper as all service assertion wrappers inherits from it. That being said, I don't see what we gain from that move. Sure, it would make sense that code to assert services are in classes designed to perform these computations (which is the only use case of that function so far). But, if somebody wants to use that function elsewhere for some quick debugging without the instantiating a LocalServiceAssertionWrapper, then having it global makes sense.

Reply:
Anyway https://www.youtube.com/watch?v=CxK_nA2iVXw

Reply:
@vincent-antaki Haha! Well I think we should avoid having too much names in base.py to avoid polluting this file's namespace as much as possible. I suggest that the method could be static on the wrapper or something related to where it's used to avoid making it too globally available.